slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -141,6 +141,15 @@ class SlideToActView @JvmOverloads constructor(
             invalidate()
         }
 
+    /** Custom Icon */
+    var icon: Int = R.drawable.slidetoact_ic_arrow
+        set(value) {
```

Comment:
I'm afraid this custom setter is causing a parsing of `VectorDrawable` twice (once for the field initialiser and once again during ctoring).

Ideally we can change it with something like:
```kotlin
/** Custom Slider Icon */
    var sliderIcon : Int = 0
        set(value) {
            field = value
            if (field != 0) {
                mDrawableArrow = parseVectorDrawableCompat(context.resources, value, context.theme)
                mDrawableArrow.setTint(iconColor)
                invalidate()
            }
        }
```

Reply:
Good catch. I've added the check in my latest commit.